### ISSUE TITLE
How to display cordinates on a map ?

### ISSUE THREAD

**Issue by LaurentGrenet** [sylikc/jpegview#59]
There is in JPEGView.ini an entry GPSMapProvider (URL for the map provider to display GPS coordinates on a map).
I set it with adequate URL.

But now, how to take advantage of it, and open the map ? I cannot find any entry in the menu ?

Thanks for your help / explanation !

**Comment by sylikc** [comment]
The coordinates only show a map link when the image has embedded GPS coordinates.  Are you viewing images with embedded GPS coords?

**Comment by LaurentGrenet** [comment]
Yes, of course. I'm talking about image embedding GPS coordinates. But I cannot see any link anywhere to show the map.
Where should be this link ?

Ooops : I just found it ! It's the coordinates themselves (in the EXIF info box) that are the link !

Maybe, my suggestion would be to display these coordinates with a color, or underlined, or any mean to make visible that they are a link...
Or (IMHO the best, because wouldn't require to display EXIF infos box) to have a entry in the "right-click" menu, ideally active only if coordinates are present, and else greyed...

**Comment by LaurentGrenet** [comment]
Here is how it looks like with full default options.
Not really visible it's a link...
![2022-05-05 18_58_56-L'Oriol de Queyrières - 17 Août 2021](https://user-images.githubusercontent.com/32399301/166974726-c92cbb68-323f-46ee-9e34-9a902932c10e.png)
.

**Comment by sylikc** [comment]
Yeah... I need to take a look at the defaults.  Mine is highlighted green on a green box in my customized INI lol

**Comment by leehow1988** [comment]
yes, it works.  ver 1.2.45.0
![jpegview_show_gps_in_map](https://user-images.githubusercontent.com/6386894/231741263-e5b24055-8cb0-41e6-b0d4-a3787f90e278.gif)

**Comment by LaurentGrenet** [comment]
@leehow1988 
Yes, the link is there and it works.
But the issue is (only...) that it is not visible "at a glance" that it's a link : It is actually displayed the same color and same font than other elements, which are not links. It's only when the cursor is moved on it that its color changes.
It would be more "obvious" if it was always displayed in another color, or another font.

**Comment by leehow1988** [comment]
Yes, it seems that I was just happy to find the location information and didn't read the description carefully.  😆
